Resources/keybinds.yml

```
@@ -1,4 +1,4 @@
-﻿version: 1 # Not used right now, whatever.
+version: 1 # Not used right now, whatever.
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)